posthog/ai/utils.py

```diff
@@ -29,6 +29,35 @@ def get_model_params(kwargs: Dict[str, Any]) -> Dict[str, Any]:
     return model_params
 
 
+def extract_core_model_params(kwargs: Dict[str, Any], provider: str) -> Dict[str, Any]:
```

Comment:
Let's do this in the plugin server, so that we only have to update one place to cover all SDKs!